proto/stride/stakeibc/host_zone.proto

```diff
@@ -7,6 +7,19 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/Stride-Labs/stride/v19/x/stakeibc/types";
 
+// CommunityPoolRebate stores the size of the community pool liquid stake
+// and the rebate percentage
+message CommunityPoolRebate {
+  string rebate_percentage = 1 [
```

Comment:
As I have continued to think about the whole design, I think it would be much simpler to not have this additional struct, but just have the rebate_percentage which will be 0 by default on the hostzone struct.  

The liquid stake amount should not remain fixed because the amount of stDYDX in the comm pool won't remain fixed -- we should epochly ICQ to measure the current amount in the comm pool and do computations on the callback of that ICQ.  This should be simpler than having to update this structure by hand using txs when they vote to stake more or choose to sell off part of their holdings and it eliminates this whole message type.

Reply:
Nevermind, ignore this comment.  Discussion on slack concluded this might be a fast follow but not needed in this first pass.